src/contract-deployment/config.ts

```diff
@@ -170,5 +171,17 @@ export const makeContractDeployConfig = async (
       factory: getContractFactory('OVM_ETH'),
       params: [config.ethConfig.initialAmount, 'Ether', 18, 'ETH'],
     },
+    'OVM_ChainStorageContainer:CTC:batches': {
```

Comment:
Add these fields to the deploy step so we'll have the storage containers ready to go.